src/Agent.Plugins/Agent.Plugins.csproj

```diff
@@ -20,6 +20,7 @@
     <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="4.4.0" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="4.4.0" /> -->
     <PackageReference Include="vss-api-netcore" Version="0.5.95-private" />
+    <PackageReference Include="azuredevops-testresultparser" Version="0.0.2" />
```

Comment:
did you sign this binary?

Reply:
I have signed with the certificate which we setup. I believe we need to resign when we release agent as a standard process

Reply:
what cert you use to sign? why we have to sign it again? are you talking about strong name validation?

Reply:
![30abe3ef-0bdf-49da-a58c-9e2dd69a3335](https://user-images.githubusercontent.com/15208679/51883287-43c56a80-23a8-11e9-9547-a89dab3b99de.png)

Reply:
that is not what you want, you should have something like
```yaml
  - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
    inputs:
      ConnectedServiceName: VSTSAgentESRP
      FolderPath: '_layout\bin'
      Pattern: |
        Agent.*.dll
        Agent.*.exe
        Microsoft.VisualStudio.Services.Agent.dll
        AgentService.exe
      UseMinimatch: true
      signConfigType: inlineSignParams
      inlineOperation: |
        [{
        "keyCode": "CP-230012",
        "operationSetCode": "SigntoolSign",
        "parameters": [{
        "parameterName": "OpusName",
        "parameterValue": "Microsoft"
        },
        {
        "parameterName": "OpusInfo",
        "parameterValue": "http://www.microsoft.com"
        },
        {
        "parameterName": "FileDigest",
        "parameterValue": "/fd \"SHA256\""
        },
        {
        "parameterName": "PageHash",
        "parameterValue": "/NPH"
        },
        {
        "parameterName": "TimeStamp",
        "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
        }
        ],
        "toolName": "sign",
        "toolVersion": "1.0"
        },
        {
        "keyCode": "CP-230012",
        "operationSetCode": "SigntoolVerify",
        "parameters": [],
        "toolName": "sign",
        "toolVersion": "1.0"
        }
        ]
    displayName: Sign Agent Assemblies (Authenticode Signing)
```

Reply:
Fixed it. Published new package with Code Sign

![image](https://user-images.githubusercontent.com/15208679/51887750-cfe08d80-23ba-11e9-8f8b-a3d3319a5fc7.png)

Reply:
@nigurr what's the code sign signature now? the screenshot still shows it's a Code Sign Test cert.